hassio/src/addon-view/config/hassio-addon-audio.ts

```diff
@@ -23,6 +23,7 @@ import {
   fetchHassioHardwareAudio,
   HassioHardwareAudioDevice,
 } from "../../../../src/data/hassio/hardware";
+import { suggestRestart } from "../../dialogs/suggestRestart";
```

Comment:
Should the const and file name be `suggestAddonRestart` instead?

Reply:
Sounds good